### DISCUSSION TITLE
Tags - QuodLibet-Puddleduck

### DISCUSSION THREAD

**Discussion by barraclm** [quodlibet/quodlibet#4498] [General]
I would like to be able to use Puddleduck to manage tags in my Audio library. I have Puddleduck installed and all the existing tags/data in the library show up. If I then add some data in Puddleduck and save, I can then see the new tag values in QuodLibet. If I save some more values using Puddleduck (same tags, different tracks) they do not show up in Quodlibet, even after deleting and recreating the library in QuodLibet. However, the new values persist as they can be viewed after closing and re-opening Puddleduck and can be seen using other tagging software, eg EasyTag. Can anyone help me understand why this is happening and how I can get round the issue, please

**Comment by declension** [comment]
Bit hard to say without concrete examples, and knowing what Puddleduck is / is doing. Perhaps it's a format thing, but you don't mention formats so hard to say.
